rave-compiler/src/main/java/com/uber/rave/compiler/CompilerUtils.java

```diff
@@ -47,11 +47,11 @@
 import javax.lang.model.util.Types;
 
 /**
- * Utitlies class for annotations.
+ * Utilities class for annotations.
  */
 final class CompilerUtils {
 
-    private static final List<Class<? extends Annotation>> SUPPORTED_ANNOTATIONS = ImmutableList.of(
+    private static final List<Class<? extends Annotation>> SUPPORT_ANNOTATIONS = ImmutableList.of(
             NonNull.class,
             Nullable.class,
```

Comment:
This is still a hard dependency on the support annotations, so if you make it `compileOnly` , it will crash the consumer project's javac if they do not include support annotations